CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Trim input values and delete properties with empty string when user record save. Refs UIU-2049.
 * Update username field validation to trim leading and trailing spaces. Refs UIU-3099.
 * Fix "Total paid amount" value that set as "$NaN" on "Refund fee/fine" modal. Refs UIU-3094.
+* Handle invalid image URLs when uploading profile photo via External URL. Refs UIU-3080.
```

Comment:
Could you reword this to something like "Validate image url provided as external url for user profile picture"?